### ISSUE TITLE
Total Recovered data for US has jumped to 0

### ISSUE THREAD

**Issue by NateEag** [CSSEGISandData/COVID-19#987]
On the site right now, when I select "US" in the "Confirmed Cases By Country/Region/Sovereignty"  panel, in the "Total Recovered" panel, I get zero. That wasn't the case as of yesterday (or even this morning, I think).

Using Chrome's dev inspector, I noticed that the JSON from [this link](https://services9.arcgis.com/N9p5hsImWXAccRNI/arcgis/rest/services/Z7biAeD8PAkqgmWhxG2A/FeatureServer/1/query?f=json&where=(Confirmed%20%3E%200)%20AND%20(Country_Region%3D%27US%27)&returnGeometry=false&spatialRel=esriSpatialRelIntersects&outFields=*&orderByFields=Confirmed%20desc%2CCountry_Region%20asc%2CProvince_State%20asc&outSR=102100&resultOffset=0&resultRecordCount=250&cacheHint=true) has 
 `0` for every `Recovered` entry.

Quick check in a bash prompt with [jq](https://stedolan.github.io/jq/) after pasting the JSON into `data.json`:

```bash
$ jq .features[].attributes.Recovered data.json | sort | uniq
0
```

**Comment by whalebonelanding** [comment]
I am experiencing the same issue with the US data.

**Comment by Sam-Gram** [comment]
Same here, I was following WA state quite closely as my parents still live there and there was one recovered person there and now there are none.

**Comment by brloynd** [comment]
Ya I am seeing this too. I know for a fact there several people that are recovered.  Heck I think half the Jazz now is considered recovered.

**Comment by aheib987** [comment]
Looks like they made another switch in their reporting.  Now it is by country roll-up rather than state for recovered count. Can someone validate this?

**Comment by ballcoach12** [comment]
This data really needs to be made consistent.

**Comment by aheib987** [comment]
Agreed, I had the US dashboard ready to publish and then I saw the inconsistency.  Hopefully they don't switch up the country field. https://public.tableau.com/profile/tony1082#!/vizhome/COVID-19_15845105291080/COVID-19

**Comment by whalebonelanding** [comment]
data are being used for decision making and need to be validated. How do we get attention? I also emailed the help desk directly. 

Best Regards, 
Patrice

**Comment by NateEag** [comment]
I think this is a consequence of this issue:

https://github.com/CSSEGISandData/COVID-19/issues/1250

Apparently recovered is no longer part of the dataset.

Feel like they should have announced this on the website, so people who were watching the recovery rates would know why they vanished.

**Comment by whalebonelanding** [comment]
Thank you Nate. I was concerned about the lack of an update on this. I wonder if the number recovered at the country level
Is valid?

Best Regards, 
Patrice

**Comment by NateEag** [comment]
I have no knowledge on that point, but if they gave up on recovery numbers as a whole, I wouldn't personally trust any that remain in the dataset.

**Comment by whalebonelanding** [comment]
I was looking for their data dictionary if one exists.

Best Regards, 
Patrice
